src/ion-autocomplete.css

```diff
@@ -5,7 +5,7 @@
     bottom: 0;
     left: 0;
     z-index: 20;
-    display: none;
+    /*display: none;*/
```

Comment:
Please delete it right away as commented out code is something you don't need ;).